### DISCUSSION TITLE
In-device search from the lock screen.

### DISCUSSION THREAD

**Discussion by PaduaPlay** [LawnchairLauncher/lawnchair#4349] [Ideas]
Having an in device search bar for apps, contacts, files etc. (with or without generic web search) would be a welcome, very nice to have, feature.

There are some search apps (not Google) such as Sesame, KISS etc. that could be useful for such integration without needing much code on the LawnChair app.

Thanks for considering.

**Comment by SuperDragonXD** [comment]
Hello! I know that this reply is a bit late, but we now support this in Lawnchair 14+. However, we don't (and probably won't) integrate with the lock screen, as it is out of scope for a home screen app.
